internal/webhook/v1/aerospikecluster_validating_webhook.go

```diff
@@ -517,6 +555,45 @@ func validateClientCertSpec(cluster *asdbv1.AerospikeCluster) error {
 	return nil
 }
 
+func validateAdminCerts(cluster *asdbv1.AerospikeCluster) error {
+	adminUser := asdbv1.GetAdminUserFromSpec(&cluster.Spec)
+	if adminUser == nil || adminUser.AerospikeAuthMode != asdbv1.AerospikeAuthModePKIOnly {
+		return nil
+	}
+
+	clientCertSpec := cluster.Spec.OperatorClientCertSpec
+
+	var certBytes []byte
+
+	var err error
+
+	if clientCertSpec == nil || clientCertSpec.CertPathInOperator == nil ||
+		clientCertSpec.CertPathInOperator.ClientCertPath == "" {
+		return nil
+	}
+
+	certBytes, err = os.ReadFile(clientCertSpec.CertPathInOperator.ClientCertPath)
+	if err != nil {
+		return fmt.Errorf("failed to read operator client cert file: %v", err)
+	}
+
+	block, _ := pem.Decode(certBytes)
+	if block == nil {
+		return fmt.Errorf("failed to decode PEM block from operator client cert file")
+	}
+
+	cert, err := x509.ParseCertificate(block.Bytes)
+	if err != nil {
+		return fmt.Errorf("failed to parse operator client certificate: %v", err)
+	}
+
+	if cert.Subject.CommonName != "admin" {
```

Comment:
1. From our samples/secrets, which file should have common name as "admin"?
2. Can we have a sample yaml to. create a cluster using pki in our config/samples folder?